tests/log_fidelity/test_bgp_shutdown.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import pytest
+
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+
+logger=logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+expected_log = "admin state is set to 'down'"
+defined_action = "sudo config bgp shutdown all"
+
+def test_bgp_shutdown(duthosts, rand_one_dut_hostname):
+    duthost=duthosts[rand_one_dut_hostname]
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="bgp_shutdown")
+    loganalyzer.expect_regex=[]
+
+    try:
+        loganalyzer.expect_regex.append(expected_log)
+        marker=loganalyzer.init()
+        duthost.command(defined_action)
```

Comment:
If the test turns down BGP, we should have a teardown implemented that restores the DUT to pretest state (BGP sessions established). (Implement a fixture?)
Similarly, should you not check if the BGP sessions are infact UP, so that after issuing BgP shutdown, you can verify if the desired string shows up in syslog.